feat(content) Remnant: Celebration of Remembrance with Taely

Zitchas opened the pull request:
## Summary
This adds a single mission wherein the player is invited to take part in a Remnant "Celebration of Remembrance" with Taely. This celebration is portrayed as a routine event that involves a significant amount of singing, dancing, and drumming. It includes both the option to simply experience it directly; or to have Taely provide some running commentary. In both cases she answers questions at the end.

Some notable details:
- This instance is led by an elderly lady, but can be led by almost anyone.
- Is often emotional for people.
- Is very freeform: It has a traditional pattern, but is expected to vary each time.
- The most serious crimes among the Remnant are those against society and acting against the collective wellbeing of the community.

## Save File
This save file can be used to play through the new mission content:
[Janice Testerv03~3016-08-05.txt](https://github.com/endless-sky/endless-sky/files/9227104/Janice.Testerv03.3016-08-05.txt)
Just take off and land, then visit the spaceport. You can ignore the currently active jobs/missions.

## Artwork Checklist
 - ~~[X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified~~
 - ~~[X] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}~~
 - ~~[X] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}~~

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That seemed both well rehearsed and very disorganized."`
+					goto celebrationrehearsed
+			label askcrying
+			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`
```
```suggestion
			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			choice
+				`	"Yes."`
+					accept
+				`	"No."`
```
The option to reject a mission is usually less blunt than this.
```suggestion
				`	"Sorry, I'm not headed that way."`
```

roadrunner56 reviewed (changes requested):
Nice worldbuilding mission!

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+					goto ceremonyleader
+				`	"That seemed both well rehearsed and very disorganized."`
+			label celebrationrehearsed
+			`	Taely smiles at that. "Well, it is and it is not. We hold these several times a month, with the expectation that people attend no more than one of them. So most of us have attended hundreds of these. Although as far as that goes, every one of these celebrations is different. Each person's participation is up to them, and so the details and feel of each celebration are unique every time." She ...
```
That is meant to be no more than one per month, so a dozen or so per year. Works out to about 120/decade. Over the course of the average human life could easily surpass a thousand.

Zitchas replied:
Alright, this has been edited to: 

> "Well, it is and it is not. We hold these several times a month, with most people taking part in one every few weeks. So most of us have attended hundreds of these. 

The point was to convey three ideas here:

1. These happen on a regular basis, several times a month.
2. Most people attend these, but not every one of them.
3. These are not formal "everyone is required to attend this" obligations.

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			choice
+				`	(Look around the room.)`
+				`	(Move in time with the beat.)`
+			`	The beat echoes in the cavern, but you quickly manage to locate the source: An elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
```
```suggestion
			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	On the final word the chant ends and dozens of deep voices join in with a roaring crescendo that builds then fades away as many new voices join in with tones of wonder. At this point, it seems that almost everyone in the room is either drumming, dancing, or singing - and many seem to be doing several simultaneously.`
+			`	As the stars wheel outside the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion of the singing is clear. The early tone of anticipation shifted to the roar of excitement then on to a peaceful tone of wonder. As the ceremony continues, this shifts to a steady, almost mechanical feel, w ...
+			choice
+				`	Stand still and absorb the moment.`
```
```suggestion
				`	(Stand still and absorb the moment.)`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	As the stars wheel outside the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion of the singing is clear. The early tone of anticipation shifted to the roar of excitement then on to a peaceful tone of wonder. As the ceremony continues, this shifts to a steady, almost mechanical feel, w ...
+			choice
+				`	Stand still and absorb the moment.`
+				`	Look around to see what is happening next.`
```
```suggestion
				`	(Look around to see what is happening next.)`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			choice
+				`	Stand still and absorb the moment.`
+				`	Look around to see what is happening next.`
+			`	Around you, everyone stands still. Most look exhausted, some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the ca ...
```
```suggestion
			`	Around you, everyone stands still. Most look exhausted, and some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the cave is nearly empty.`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	(Look around the room.)`
+				`	(Move in time with the beat.)`
+			`	At your side, Taely gestures again. "Existence starts with the patterns of the universe, and our story starts with life."`
+			`	The beat echoes in the cavern, but you quickly manage to locate the source: An elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
```
```suggestion
			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			choice
+				`	Stand still and absorb the moment.`
+				`	Look around to see what is happening next.`
+			`	Around you, everyone stands still. Most look exhausted, some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the ca ...
```
```suggestion
			`	Around you, everyone stands still. Most look exhausted, and some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the cave is nearly empty.`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That seemed both well rehearsed and very disorganized."`
+					goto celebrationrehearsed
+				`	"What constitutes your most serious crimes?"`
+			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective wellbeing is the single most destructive and senseless acts someone can do."`
```
```suggestion
			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective wellbeing is the single most destructive and senseless act someone can do."`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+	on complete
+		payment 50000
+		conversation
+			`When you arrive on Postverta there is a small group of Remnant waiting at the landing bay for you. As they accept the crystal from you one gestures "Thank you for bringing this to me, Captain <first>. Being able to join in the Remembrance will help me to remember and celebrate those who have gone on before." They head off into the depths of the station, leaving you in the bay.`
```
```suggestion
			`When you arrive on Postverta there is a small group of Remnant waiting at the landing bay for you. As they accept the crystal from you one gestures, "Thank you for bringing this to me, Captain <first>. Being able to join in the Remembrance will help me to remember and celebrate those who have gone on before." They head off into the depths of the station, leaving you in the bay.`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back an ...
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and have begun to move around in a circle. Everywhere you look there are people dancing in countless variations.`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back an ...
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and have begun to move around in a circle. Everywhere you look there are people dancing in countless variations.`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That was awe-inspiring."`
+				`	"Who leads these ceremonies?"`
+					goto ceremonyleader
+				`	"That seemed both well rehearsed and very disorganized."`
```
```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+					goto askcrying
+				`	"Who leads these ceremonies?"`
+					goto ceremonyleader
+				`	"That seemed both well rehearsed and very disorganized."`
```
```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+					goto askcrying
+				`	"Who leads these ceremonies?"`
+					goto ceremonyleader
+				`	"That seemed both well rehearsed and very disorganized."`
```
```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That was awe-inspiring."`
+					goto awestruck
+				`	"Why were some of the people crying?`
+				`	"That seemed both well rehearsed and very disorganized."`
```
```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+					goto awestruck
+				`	"Why were some of the people crying?`
+					goto askcrying
+				`	"That seemed both well rehearsed and very disorganized."`
```
```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+					goto awestruck
+				`	"Why were some of the people crying?`
+					goto askcrying
+				`	"That seemed both well rehearsed and very disorganized."`
```
```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	As the stars wheel outside the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion of the singing is clear. The early tone of anticipation shifted to the roar of excitement then on to a peaceful tone of wonder. As the ceremony continues, this shifts to a steady, almost mechanical feel, w ...
+			`	Through it all, Taely offers a running commentary on how each stretch of the ceremony represents a different piece of their collective history, ranging from the launch of the first human into space to its commercialization, and then the new age of exploration opened by the advent of hyperdrives. It also includes numerous conflicts, disasters, and wars. Much of it is familiar to you from what ...
+			choice
+				`	Stand still and absorb the moment.`
```
```suggestion
				`	(Stand still and absorb the moment.)`
```

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	Through it all, Taely offers a running commentary on how each stretch of the ceremony represents a different piece of their collective history, ranging from the launch of the first human into space to its commercialization, and then the new age of exploration opened by the advent of hyperdrives. It also includes numerous conflicts, disasters, and wars. Much of it is familiar to you from what ...
+			choice
+				`	Stand still and absorb the moment.`
+				`	Look around to see what is happening next.`
```
```suggestion
				`	(Look around to see what is happening next.)`
```

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+					goto ceremonyleader
+				`	"That seemed both well-rehearsed and very disorganized."`
+			label celebrationrehearsed
+			`	Taely smiles at that. "Well, it is and it is not. We hold these several times a month, with most people taking part in one every few weeks. So most of us have attended hundreds of these. Although as far as that goes, every one of these celebrations is different. Each person's participation is up to them, and so the details and feel of each celebration are unique every time." She makes a gest ...
```
What she said earlier made it seem like this was a monthly thing.

Zitchas replied:
It is, sort of. The event itself takes place many times per month, but it is generally considered to be something that a person only attends on roughly monthly basis. Enough of these take place in a month that all the Remnant present on that world would be able to attend. There's probably quite a few going on any particular day.

This is sort of like Friday religious services (or Saturday, or Sunday, depending on one's religion of choice); except that the cultural expectation is of monthly attendance, not weekly.

On a more subtle level, this is intended to show that yes, there is religion here; but it's much more casual and informal than many real-life present-day religious denominations.

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+				`	"You said 'virtually.' Who is omitted?"`
+			`	Taely scowls a bit at this. "Only those of guilty of the most serious crimes cannot stand as a lead." She pauses, and continues with an amused tone. "Even children take the lead now and then, usually with a parent or sibling to guide them." She smiles at that, then gestures for you to continue asking your questions.`
```
```suggestion
			`	Taely scowls a bit at this. "Only those guilty of the most serious crimes cannot stand as a lead." She pauses, and continues with an amused tone. "Even children take the lead now and then, usually with a parent or sibling to guide them." She smiles at that, then gestures for you to continue asking your questions.`
```

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+			label askcrying
+			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`
```
Is this the *only* reason? This whole seemed like a pretty emotional experience. I'd imagine some people would cry of joy as well, and both crying of joy and sorrow could be for a variety of reasons.

Zitchas replied:
There is, in all likelyhood, a variety of reasons. It is a fairly emotional thing. I didn't really want to expound at length on all the many different ways there might be someone crying.

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+			label celebrationrequest
+			`	Taely strides over to a console near the door and retrieves a crystal from a compartment within the console. She slips it into a case and turns to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present can experience it too. This recording is ...
```
```suggestion
			`	Taely strides over to a console near the door and retrieves a crystal from a compartment within the console. She slips it into a case and turns to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present can experience it too. This recording is destined for those working in Postverta, as a friend of one of those being mourned today is stationed there. Could you deliver this to them please?"`
```

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+					goto celebrationrehearsed
+			label celebrationrequest
+			`	Taely strides over to a console near the door and retrieves a crystal from a compartment within the console. She slips it into a case and turns to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present can experience it too. This recording is ...
+			action set "Remnant: Attended Celebration of Remembrance with Taely"
```
Does this work?

Zitchas replied:
It did not. Now it does. Thanks for asking!

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+		payment 50000
+		conversation
+			`When you arrive on Postverta there is a small group of Remnant waiting at the landing bay for you. As they accept the crystal from you one gestures, "Thank you for bringing this to me, Captain <first>. Being able to join in the Remembrance will help me to remember and celebrate those who have gone on before." They head off into the depths of the station, leaving you in the bay.`
+			`	Taely never mentioned anything about compensation, but as soon as the Remnant left your commlink notifies you that you have been assigned <payment>.`
```
```suggestion
			`	Taely never mentioned anything about compensation, but as soon as the Remnant leave, your commlink notifies you that you have been assigned <payment>.`
```
```suggestion
			`	Taely never mentioned anything about compensation, but as soon as the Remnant have left, your commlink notifies you that you have been assigned <payment>.`
```

warp-core reviewed (commented):
These labels seem unnecessarily bulky. `accept`, `decline`, `explain`, `end`, `annotated` (or `annotate`), `request`, `awe`, `crying`, and `leader` alone would be fine.

It also occurs to me now that this is a spaceport mission so the player wouldn't just be disembarking their ship when they see the crowd of Remnant going somewhere. I still think that it is presented in an odd way, even if we are just entering the spaceport, not exiting our ship.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"Yes, that would be appreciated."`
+					goto celebrationannotated
+			`	The flow of people you are a part of joins with other streams of people, finally leading out of the typical hallways and into a large cave. A large opening on one side opens upwards towards the night sky, framed by rough stone and a handful of shield emitters. Stalagmites ring the perimeter of the room with matching stalactites above them, but there is no trace of drips or moisture here. Whi ...
+			`	Without warning the room stills, and the doors close. You see a flurry of gestures wash over the room, as if everyone is repeating what someone closer to the center is saying, but you cannot see anyone starting it. "Perimeter secure?" it asks, followed by a ripple of gestures from the edges of the room back toward the center: "All clear."`
```
Nope, this is deliberate. The protagonist is not aware of any noise or obvious reason for the room to still. It is intended to be incongruous and something that the player might take note of.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			choice
+				`	(Look around the room.)`
+				`	(Move in time with the beat.)`
+			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	"Thump-thump."`
```
It is intended to remind a person of a heartbeat.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back an ...
```
The circle as a whole is moving in place, as in the circle doesn't change location, just the people who are a part of it moving around the circle.

warp-core replied:
Then, I would expect "moving in a circle" or perhaps "moving around a circle".

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	Taely notices your anxiety and gestures reassuringly. "All is well."`
+			label countdown
+			`	On the final word the chant ends and dozens of deep voices join in with a roaring crescendo that builds then fades away as many new voices join in with tones of wonder. At this point, it seems that almost everyone in the room is either drumming, dancing, or singing - and many seem to be doing several simultaneously.`
+			`	As the stars wheel outside the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion of the singing is clear. The early tone of anticipation shifted to the roar of excitement then on to a peaceful tone of wonder. As the ceremony continues, this shifts to a steady, almost mechanical feel, w ...
```
It is intended to evoke the passage of time. This part of the description steps  back from an actual description of the sequence of events to a larger "big picture" overview of the event.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+		"day" = 7
+	on offer
+		conversation
+			`You stop by the starport just in time to see a large number of Remnant going somewhere together. A few of them notice you and gesture an invitation to accompany them, but before you can react Taely appears from the stream of people and approaches you.`
```
This is intended to be fairly fast sequence. I don't personally feel that I am frozen in place while Taely shows up, but I can see about tweaking that.

EjoThims commented:
That annotated section seems like a great candidate for https://github.com/endless-sky/endless-sky/pull/5440 instead of doubling up all that text.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	"Thump."`
+			choice
+				`	(Look around the room.)`
+				`	(Move in time with the beat.)`
```
Correct. This is both a break in the flow and an opportunity to let the player do something. Neither option affects the story, though. Which is logical, since the player isn't the focus of what is going on, and has no bearing on the events. As such, this is more an RP detail so that players can decide for themselves if they're just observing or actually taking part.

Zitchas commented:
Yes, it would be. Once that's merged I'd certainly be willing to see about using it to streamline the text; but I don't want to hold this up just for that.

EjoThims commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+			label askcrying
+			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`
```
```suggestion
			`	Taely gestures gently. "Many were friends of the crew of a ship that was recently destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals. Sometimes the whole experience is simply overwhelming, and some of us still sorrow for other losses in our stories."`
```

EjoThims commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+			label celebrationrequest
+			`	Taely strides over to a console near the door and retrieves a crystal from a compartment within the console. She slips it into a case and turns to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present can experience it too. This recording is ...
```
```suggestion
			`	Taely strides over to a console near the closest door and retrieves a crystal from a compartment within. She slips it into a case and turns back to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present at a particular event may still experience it too. This recording is destined for those working in Postverta, as a friend of one of those being mourned today is stationed there. Could you deliver this to them please?"`
```

EjoThims commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+	on complete
+		payment 50000
+		conversation
+			`When you arrive on Postverta there is a small group of Remnant waiting at the landing bay for you. As they accept the crystal from you one gestures, "Thank you for bringing this to me, Captain <first>. Being able to join in the Remembrance will help me to remember and celebrate those who have gone on before." They head off into the depths of the station, leaving you in the bay.`
```
```suggestion
			`When you arrive on Postverta there is a small group of Remnant waiting at the landing bay for you. As you hand the crystal to one, another gestures, "Thank you for bringing this to us, Captain <first>. Being able to join in the Remembrance will help me to celebrate those who have gone on before." The group heads off into the depths of the station, leaving you in the bay.`
```

EjoThims reviewed (changes requested):
Some tone/meaning changes to hopefully address the discussion that's already gone on, and some more general ideas/comments about other things that stick out.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	(Move in time with the beat.)`
+			`	At your side, Taely gestures again. "Existence starts with the patterns of the universe, and our story starts with life."`
+			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
+			`	"Thump-thump."`
```
I don't have time to type out the addition part right now, but will tomorrow about this time. So I'm leaving the suggestions that need me to do additional input un-committed so as to remind myself to do them tomorrow.

W1zrad replied:
See above - I personally prefer it as-is.

Zitchas replied:
Likewise, I think I prefer to leave this as-is, as per W1zrad's comments.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"Who leads these ceremonies?"`
+			label ceremonyleader
+			`	Taely gestures take on a tone of conviction. "For something like this, or anything of a social or spiritual nature for that matter, we determine who takes the lead for various parts at the moment based on who is present. Whether we are talking about our connection with society, or our connection to the Universe, we must each forge our own understanding of it. As such, when we gather together ...
+			`	"As a result, each celebration takes on a different feel - after all, each of us has our own unique perspective, and bring our own experiences to contribute." She pauses, and adopts a respectful tone. "The ceremony is not just about major events, either. Everyone who takes part brings their own life experiences and memories to it. Memories of friends and family who are no longer with us are  ...
```
> Memories of friends and family who are no longer with us are a common as well.

That "a" near the end doesn't seem right.

EjoThims replied:
I think it was originally for something like "a common sight" but decided against.

Fixed :)

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+
+mission "Remnant: Celebration 1"
+	name "Deliver Celebration records to <destination>"
+	description "The Remnant have asked you to take records of their celebration to <destination>"
```
```suggestion
	description "The Remnant have asked you to take records of their celebration to <destination>."
```

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	"No thanks."`
+					goto celebrationdeclined
+			label celebrationattendance
+			`	As the two of you join the flow of people Taely gestures once again. "If you would like, I could provide you with some commentary and explanation, but it is more about feeling than doing anything specific. Together, we remember and celebrate through free-form dance mixed with singing and music. You are welcome to participate, but try not to disrupt or impede others in the way you join the da ...
```
```suggestion
			`	As the two of you join the flow of people, Taely gestures once again. "If you would like, I could provide you with some commentary and explanation, but it is more about feeling than doing anything specific. Together, we remember and celebrate through free-form dance mixed with singing and music. You are welcome to participate, but try not to disrupt or impede others in the way you join the dancing."`
```

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	"Thump."`
+			`	"Thump."`
+			`	"Thump."`
+			`	"Thump."`
```
I'm cool with the thumping, but I don't quite like the progression idea. There are only a few variations we can do (in fact you listed them: `thump, Thump, THUMP`). Unless there are literally exactly three "thumps" happening, I think the starkness of the word change takes away from the reader's personal ability to interpret "gradualness", if that makes any sense.
Basically, each word change (`thump -> Thump`) would feel more like a leap rather than a slower, more gradual increase to me.

Zitchas replied:
Given W1zrad's comments, I prefer to leave this as is. While there is progression here; it is far more than just three thumps. This is more of a crescendo over probably at least a dozen thumps.

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				`	(Look around to see what is happening next.)`
+			`	Around you, everyone has gone still. Most look exhausted, and some are sitting on the floor or leaning against the walls. A few people look like they have been crying, and those nearby are hugging them or simply touching their shoulder. Most of the people you can see have reverent and joyful expressions. After the silence lingers for what feels like several minutes, the doors open and people ...
+			label celebrationend
+			`	"I always find these Celebrations of Remembrance to be invigorating. What did you think of it, <first>?"`
```
```suggestion
			`	"I always find these Celebrations of Remembrance to be invigorating," Taely says. "What did you think of it, <first>?"`
```

W1zrad reviewed (changes requested):
I'd say this is probably some of your best writing yet. Very nice stuff! Just a handful of thoughts/edits.

EjoThims reviewed (changes requested):
Couple of little things me and Wiz both missed, and a couple possible tone enhancements.

Don't forget to copy everything up to the non-annotated section either ;)

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+	to offer
+		has "event: remnant: cognizance calm"
+		has "license: Remnant Capital"
+		# random < 50
```
So no one forget about this.
```suggestion
		random < 50
```

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			label celebrationdefinition
+			`	"The Celebration of Remembrance is a gathering; we take a moment to remember where we have come from, appreciate where we are, and express hope for where we will be in the future." Her gestures take on a critical tone. "That sounds very formal, but it is more artistic than defined." She glances at the slowing flow of people. "It will be starting shortly, and I want to attend this month, so I ...
+			choice
+				`	"Yes, I do."`
```
"Yes, I do [like to join you.]"
That doesn't sound right.
```suggestion
				`	"Yes, I would."`
```

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+					goto celebrationannotated
+			`	The flow of people you are a part of joins with other streams, finally leading out of the typical hallways and into a large cave. A large opening on one side opens upwards towards the night sky, framed by rough stone and a handful of shield emitters. Stalagmites ring the perimeter of the room with matching stalactites above them, but there is no trace of drips or moisture here. While the wal ...
+			`	Without warning the room stills, and the doors close. You see a flurry of gestures wash over the room, as if everyone is repeating what someone closer to the center is saying, but you cannot see anyone starting it. "Perimeter secure?" it asks, followed by a ripple of gestures from the edges of the room back toward the center: "All clear."`
+			`	Suddenly you are aware of a drum beat building in the silence, and realize that it had started with the first ripple of gestures, so quiet that it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
```
```suggestion
			`	Suddenly, you are aware of a drum beat building in the silence, and realize that it had started with the first ripple of gestures, so quiet that it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
```

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			choice
+				`	(Look around the room.)`
+				`	(Move in time with the beat.)`
+			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
```
Are hollow stalagmites a thing in real life?
It seems hollow stalactites are a thing, but not so much stalagmites?
Is the implication that the dripping that usually forms these happened upwards, for some reason, I guess someone was screwing with gravity?
Or, something else? Are they just a different type to what we might have on Earth? Are they supposed to be artificial?

Zitchas replied:
Go with the simplest explanation: They're artificially hollowed out. I'm not actually averse to the other options (such as someone messing with gravity or that they are somehow naturally like that), but simply that they've been hollowed out artificially somehow is the main thing.

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back an ...
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving around in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving around in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`
```

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually coming out of the typical hallways and into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or trac ...
+			`	Taely gestures quietly to you. "We start with the ritual query and response asking if we are safe. Safe in this moment, at least."`
+			`	Without warning the room stills, and all the doors close. A flurry of gestures washes out from the center of the room, as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`
+			`	Suddenly you become aware of a drum beat building in the silence, and then realize that it had already started with the first ripple of gestures, so quiet that, until now, it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
```
```suggestion
			`	Suddenly, you become aware of a drum beat building in the silence, and then realize that it had already started with the first ripple of gestures, so quiet that, until now, it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
```

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	"Thump-thump."`
+			`	Looking around the room you can see people starting to move in time with the beat. Some are already swaying their whole body while others are just nodding their heads or tapping their feet. Standing next to you, Taely has a hand over her heart, tapping softly in time. Soon you hear other drummers join in and the beat resonates through the room. Now people are not merely swaying to the beat,  ...
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving around in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`
```

warp-core reviewed (commented):
I still stand by some of my comments from my earlier review, but it seems no one agrees and I don't think there's anything I can add to them.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			label countdownannotated
+			`	On the final word the chant ends and dozens of deep voices join in with a roaring crescendo that builds then fades away as many new voices join in with tones of wonder. At this point, it seems that almost everyone in the room is either drumming, dancing, or singing - and many seem to be doing several simultaneously.`
+			`	As the stars wheel outside the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion of the singing is clear. The early tone of anticipation shifted to the roar of excitement then on to a peaceful tone of wonder. As the ceremony continues, this shifts to a steady, almost mechanical feel, w ...
+			`	Through it all, Taely offers a running commentary on how each stretch of the ceremony represents a different piece of their collective history, ranging from the launch of the first human into space to its commercialization, and then the new age of exploration opened by the advent of hyperdrives. It also includes numerous conflicts, disasters, and wars. Much of it is familiar to you from what ...
```
There are now fewer discoveries. lol.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			label countdownannotated
+			`	On the final word the chant ends and dozens of deep voices join in with a roaring crescendo that builds then fades away as many new voices join in with tones of wonder. At this point, it seems that almost everyone in the room is either drumming, dancing, or singing - and many seem to be doing several simultaneously.`
+			`	As the stars wheel outside the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion of the singing is clear. The early tone of anticipation shifted to the roar of excitement then on to a peaceful tone of wonder. As the ceremony continues, this shifts to a steady, almost mechanical feel, w ...
+			`	Through it all, Taely offers a running commentary on how each stretch of the ceremony represents a different piece of their collective history, ranging from the launch of the first human into space to its commercialization, and then the new age of exploration opened by the advent of hyperdrives. It also includes numerous conflicts, disasters, and wars. Much of it is familiar to you from what ...
```
Likewise, fewer discoveries.

Zitchas commented:
@warp-core  Regarding your comment about the labels: for ease of management, navigation, using find to locate all instances of references to a particular label, etc, I tend to use two-word amalgamated labels to help ensure that they are utterly unique within the file; not just within that particular mission. It also helps to ensure I don't inadvertantly use a key word, like "accept" as a label, which also generates similar sorts of confusion.

I did also finally get around to checking that action you commented on. It wasn't working as intended, so as a result I have fixed it. Thanks for commenting on it so that I would verify that it is performing its intended action.

At this point, I think I have addressed all your comments.

tibetiroka commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+				`	"What constitutes your most serious crimes?"`
+			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective wellbeing is the single most destructive and senseless act someone can perpetrate."`
```
```suggestion
			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective well-being is the single most destructive and senseless act someone can perpetrate."`
```

TheMarksman-ES commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+				`	"No, that is alright. I will just experience it."`
+				`	"Yes, that would be appreciated."`
+					goto celebrationannotated
+			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually coming out of the typical hallways and into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or trac ...
```
```suggestion
			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually emerging from the typical hallways into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or traces of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
```

TheMarksman-ES commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+				`	"Yes, that would be appreciated."`
+					goto celebrationannotated
+			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually coming out of the typical hallways and into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or trac ...
+			`	Without warning the room stills, and all the doors close. A flurry of gestures washes out from the center of the room, as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`
```
```suggestion
			`	Without warning, the room stills, and all the doors close. A flurry of gestures washes out from the center of the room as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`
```

Zitchas replied:
Are you sure? This doesn't feel like a good flow with the comma there.

tibetiroka replied:
I think both versions are correct in terms of grammar, so it is a question of which rhythm you prefer.

warp-core replied:
"Without warning the room stills, and all the doors close."
"Warning the room stills," sounds like giving a warning to "the room stills", whatever a "room still" is.
The 'without' means no such warning was given.
There probably should be a comma there.

tibetiroka replied:
> giving a warning to "the room stills"
I'm pretty sure [still can also be a verb](https://www.oxfordlearnersdictionaries.com/definition/english/still_4), in which case the comma isn't necessary.

EjoThims replied:
I think it could go either way still, but @warp-core convinced me here it works best with the comma. :)

Zitchas replied:
I don't follow warp-core's example. The lack of warning is in regards to the room stilling. 

To me, having a comma there breaks the flow. "Without warning." Well, without warning about what?

It's not two different ideas, it's not even two different things influencing each other. It's one thing.

EjoThims replied:
Without is a modifier to the phrase, not a part of it; the comma breaks up the phrase it's modifying.

"(((Without (warning the room stills)), (and all the doors close.))"

"(((Without (warning)), (the room stills), (and all the doors close.))"

Zitchas replied:
OK, well, at this point I kind of feel it's coming down to what is required by grammatical rules.

1. If a comma is grammatically *required* here, then I put it in.
2. If a comma is *not* required, then I would prefer to not have the comma as I feel it breaks the flow and the feel of what this detail is supposed to convey.

warp-core replied:
I believe the comma is grammatically required.

Zitchas replied:
This does not feel conclusive, but the members of the writing team who have commented seem to accept without a comma, so I am leaving it out.

TheMarksman-ES commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+				`	(Look around with anticipation.)`
+					goto countdown
+				`	(Look around nervously.)`
+			`	Taely notices your anxiety and gestures reassuringly. "All is well."`
```
```suggestion
			`	Taely notices your anxiety and gestures reassuringly: "All is well."`
```

TheMarksman-ES commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+			`	Taely notices your anxiety and gestures reassuringly. "All is well."`
+			label countdown
+			`	On the final word the chant ends and dozens of deep voices join together in a roaring crescendo before fading away, replaced by many new voices expressing tones of wonder. At this point, it seems that almost every single person in the room is drumming, dancing, or singing - many seem to be doing several simultaneously.`
+			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`
```
```suggestion
			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`
```

TheMarksman-ES commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+			label countdown
+			`	On the final word the chant ends and dozens of deep voices join together in a roaring crescendo before fading away, replaced by many new voices expressing tones of wonder. At this point, it seems that almost every single person in the room is drumming, dancing, or singing - many seem to be doing several simultaneously.`
+			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`
+			`	As the ceremony continues, this shifts again, to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and struc ...
```
```suggestion
			`	As the ceremony continues, this shifts again to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other deeply emotional pairings before finally culminating in a return to the original, unified heartbeat as the singers' voices come together, reaching upwards in hope before finally everything trails off into silence.`
```

TheMarksman-ES commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+			`	Around you, everyone has gone still. Most look exhausted, and some are sitting on the floor or leaning against the walls. A few people look like they have been crying, and those nearby are hugging them or simply touching their shoulder. Most of the people you can see have reverent and joyful expressions. After the silence lingers for what feels like several minutes, the doors open and people ...
+				goto celebrationend
+			label celebrationannotated
+			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually coming out of the typical hallways and into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or trac ...
```
```suggestion
			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually emerging from the typical hallways into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or traces of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
```

TheMarksman-ES commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+			label celebrationannotated
+			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually coming out of the typical hallways and into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or trac ...
+			`	Taely gestures quietly to you. "We start with the ritual query and response asking if we are safe. Safe in this moment, at least."`
+			`	Without warning the room stills, and all the doors close. A flurry of gestures washes out from the center of the room, as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`
```
```suggestion
			`	Without warning, the room stills, and all the doors close. A flurry of gestures washes out from the center of the room as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`
```

Zitchas replied:
As per the other instance, I think 'Without warning the room stills,' sounds better as one phrase rather than two.

TheMarksman-ES commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+			label countdownannotated
+			`	On the final word the chant ends and dozens of deep voices join together in a roaring crescendo before fading away, replaced by many new voices expressing tones of wonder. At this point, it seems that almost every single person in the room is drumming, dancing, or singing - many seem to be doing several simultaneously.`
+			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`
+			`	As the ceremony continues, this shifts again, to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and struc ...
```
```suggestion
			`	As the ceremony continues, this shifts again to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other deeply emotional pairings before finally culminating in a return to the original, unified heartbeat as the singers' voices come together, reaching upwards in hope before finally everything trails off into silence.`
```

TheMarksman-ES commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+			`	Taely notices your anxiety and gestures reassuringly. "They sing of the days of humanity's infancy; when we first ventured out of the cradle."`
+			label countdownannotated
+			`	On the final word the chant ends and dozens of deep voices join together in a roaring crescendo before fading away, replaced by many new voices expressing tones of wonder. At this point, it seems that almost every single person in the room is drumming, dancing, or singing - many seem to be doing several simultaneously.`
+			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`
```
```suggestion
			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`
```

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+					goto celebrationdefinition
+				`	"No thanks."`
+			label celebrationdeclined
+			`	She nods. "Well, may the Embers guide you, <first>." With that, she rejoins the stream of people as they vanish from sight.`
```
Hmm, that makes it sound more like Taely vanishing, as opposed to the stream.

EjoThims replied:
It would, but that's a change I made when I switched the wording. Thought it would help address the awkwardness @quyykk mentioned to talk about when you could no longer see the group versus when you could no longer see her.

But suggestion from @TheMarksman-ES is improvement on original even though switching that back.

Zitchas replied:
Would it be acceptable to just leave it as-is, then?

Zitchas replied:
Alright then, leaving it as-is. Thanks!

roadrunner56 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+				`	(Look around to see what is happening next.)`
+			`	Around you, everyone has gone still. Most look exhausted, and some are sitting on the floor or leaning against the walls. A few people look like they have been crying, and those nearby are hugging them or simply touching their shoulder. Most of the people you can see have reverent and joyful expressions. After the silence lingers for what feels like several minutes, the doors open and people ...
+			label celebrationend
+			`	"I always find these Celebrations of Remembrance to be invigorating," Taely says. "What did you think of it, <first>?"`
```
```suggestion
			`	"I always find these Celebrations of Remembrance ceremonies to be invigorating," Taely says. "What did you think of it, <first>?"`
```

EjoThims replied:
If adding "ceremonies," should become "Celebration of Remembrance" instead of "Celebrations."

Zitchas replied:
I get that it is technically correct; however I think it also makes sense that Taely just refers to it as "Celebrations of Remembrance" and omits the ceremonies label/descriptor.

roadrunner56 replied:
Yeah looking back I don't know why I made this comment. Ignore it.

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -184,42 +184,43 @@ mission "Remnant: Celebration 1"
-			`	The flow of people you are a part of joins with other streams, finally leading out of the typical hallways and into a large cave. A large opening on one side opens upwards towards the night sky, framed by rough stone and a handful of shield emitters. Stalagmites ring the perimeter of the room with matching stalactites above them, but there is no trace of drips or moisture here. While the wal ...
-			`	Without warning the room stills, and the doors close. You see a flurry of gestures wash over the room, as if everyone is repeating what someone closer to the center is saying, but you cannot see anyone starting it. "Perimeter secure?" it asks, followed by a ripple of gestures from the edges of the room back toward the center: "All clear."`
-			`	Suddenly you are aware of a drum beat building in the silence, and realize that it had started with the first ripple of gestures, so quiet that it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
+			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually emerging from the typical hallways into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or traces o ...
```
```suggestion
			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually emerging from the typical hallways into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up toward the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or traces of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
```

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -184,42 +184,43 @@ mission "Remnant: Celebration 1"
-			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back an ...
-			`	Out of nowhere you here a voice singing. You cannot make out what it is saying, but the tone sounds like one of anticipation. You can, however, clearly make out a chant that cuts through the solo voice in time with the drumming:`
+			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, and some are nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others r ...
+			`	Out of the chaos you hear a voice rise in song. You cannot make out what it is saying, but it carries the tones of anticipation. You can, however, clearly decipher a chant from many other voices, cutting through the solo voice in time with the drumming but not overwhelming either:`
```
```suggestion
			`	Out of the chaos you hear a voice rise in song. You cannot make out what it is saying, but it carries the tones of anticipation. You can, however, clearly decipher a chant from many other voices, cutting through the solo voice in time with the drumming:`
```
Personal preference. That last bit feels too run-onish; I liked the original flow better.

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -243,7 +244,8 @@ mission "Remnant: Celebration 1"
 			`	On the final word the chant ends and dozens of deep voices join together in a roaring crescendo before fading away, replaced by many new voices expressing tones of wonder. At this point, it seems that almost every single person in the room is drumming, dancing, or singing - many seem to be doing several simultaneously.`
-			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder. As the ceremony continues, this shifts again, ...
+			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`
+			`	As the ceremony continues, this shifts again to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and struct ...
```
```suggestion
			`	As the ceremony continues, this shifts again to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in dual tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other deeply emotional pairings before finally culminating in a return to the original, unified heartbeat as the singers' voices come together, reaching upwards in hope before finally everything trails off into silence.`
```
`duel` with an `e` is a fight between two people. `dual` with an `a` means two things.
